src/__tests__/posthog-persistence.js

```diff
@@ -0,0 +1,30 @@
+import { PostHogPersistence } from '../posthog-persistence'
+
+given('lib', () => Object.assign(new PostHogPersistence({ name: 'bla', persistence: 'cookie' }), given.overrides))
```

Comment:
Suggestion: `given('persistence', () => new PostHogPersistence({ name: 'bla', persistence: 'cookie' }))` - no need for overrides here (it's an artifact of core being stupid complex that it's even needed), this simplifies the test at hand.